composer.json

```diff
@@ -21,7 +21,8 @@
 
 	"require-dev": {
 		"satooshi/php-coveralls": "dev-master",
-		"phpunit/php-file-iterator": "1.3.3"
+		"phpunit/php-file-iterator": "1.3.3",
+		"phpunit/phpunit": "~4.0"
```

Comment:
I go with the current, after seeing beaten to death discussions about this I really go with the group pressure :blush: